children/schema.py

```diff
@@ -125,10 +134,100 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return SubmitChildrenAndGuardianMutation(children=children, guardian=guardian)
 
 
+class UpdateChildMutation(graphene.relay.ClientIDMutation):
+    class Input:
+        id = graphene.ID(required=True)
+        first_name = graphene.String()
+        last_name = graphene.String()
+        birthdate = graphene.Date()
+        postal_code = graphene.String()
+        relationship = RelationshipInput()
+
+    id = graphene.ID()
```

Comment:
isn't it easier to just return `ChildNode`?

Reply:
Yes that would be nicer in many ways, but it would also mean that the response fields need to be nested inside another field, which I don't really like in this particular situation.

Reply:
And after some researching and rethinking, I think your way is superior, so I will switch to that :)